Documentation/ApiOverview/Backend/BroadcastChannels.rst

```diff
@@ -75,29 +75,32 @@ Example code:
    });
 
 
-Hook into :php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['typo3/backend.php']['constructPostProcess']` to load a custom
-:php:`BackendController` hook that loads the event handler, e.g. via RequireJS.
+Hook into
+:php:`$GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['typo3/backend.php']['constructPostProcess']`
+to load a custom :php:`BackendController` hook that loads the event handlers
+JavaScript.
 
 Example code:
 
-.. code-block:: php
-   :caption: EXT:some_extension/ext_localconf.php
+..  code-block:: php
+    :caption: EXT:some_extension/ext_localconf.php
 
-   $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['typo3/backend.php']['constructPostProcess'][]
-       = \Vendor\MyExtension\Hooks\BackendControllerHook::class . '->registerClientSideEventHandler';
+    $GLOBALS['TYPO3_CONF_VARS']['SC_OPTIONS']['typo3/backend.php']['constructPostProcess'][]
+        = \MyVendor\MyExtension\Hooks\BackendControllerHook::class . '->registerClientSideEventHandler';
 
 
 .. code-block:: php
-   :caption: EXT:some_extension/Classes/Hooks/BackendControllerHook.php
-
-   use TYPO3\CMS\Core\Utility\GeneralUtility;
-   use TYPO3\CMS\Core\Page\PageRenderer;
-
-   class BackendControllerHook
-   {
-       public function registerClientSideEventHandler(): void
-       {
-           $pageRenderer = GeneralUtility::makeInstance(PageRenderer::class);
-           $pageRenderer->loadRequireJsModule('TYPO3/CMS/MyExtension/EventHandler');
-       }
+    :caption: EXT:my_extension/Classes/Hooks/BackendControllerHook.php
+
+    use TYPO3\CMS\Core\Utility\GeneralUtility;
+    use TYPO3\CMS\Core\Page\PageRenderer;
+
+    class BackendControllerHook
+    {
+        public function registerClientSideEventHandler(): void
+        {
+            $pageRenderer = GeneralUtility::makeInstance(PageRenderer::class);
+            $pageRenderer->loadJavaScriptModule('@myvendor/my-extension/event-handler.js');
```

Comment:
Do we have a page which describes that JavaScript ES6 stuff (@myvendor/my-extension/event-handler.js)? So I propose to link to it in a seealso or something to guide the user what it is all about.

Reply:
Nope, not yet, still working on it: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/5

Reply:
I appreciate your work on that JS stuff, for me it is still hard to grab and understand - thumbs up from my side.
So, maybe just add a todo, so someone(TM) will stumble upon it at someday and adds the link :)

Reply:
How about we make a page stub where we add a todo to explain ES6 and then link there from all the places that need links?

Reply:
Thumbs up from me :-) It is main, so WIP. And perhaps someone who reads it want to contribute (dream).

Can we add something like: If you know about this topic please consider to contribute. 8with link to the contribution page).

Reply:
I actually already assigned myself to the issue. Contributions would be a dream but it doesn't seem like any are coming for now

Reply:
done